backend/services/store/menus/index.go

```
@@ -45,3 +53,71 @@ func (s *MenuService) GetMenusByStoreID(c context.Context, storeID uuid.UUID) ([
+	if err != nil {
+		return menus.MenuDisplay{}, err
+	}
+
+	err = s.imageRepository.PutObject(c, processedImage, entities.StoreImageObjectKey(imageObjectKey), contentType)
+	if err != nil {
+		return menus.MenuDisplay{}, services.ErrFailedToStoreImage
+	}
+
+	imageURL, err := s.imageURLGenerator.GenerateMenuImageURL(c, imageObjectKey)
+	if err != nil {
+		s.imageRepository.DeleteObject(c, entities.StoreImageObjectKey(imageObjectKey))
```

golangci-lint (2.12.2)

[error] Error return value of `s.imageRepository.DeleteObject` is not checked

(errcheck)